### PR TITLE
adding cumulative infections sa vignette

### PR DESCRIPTION
**Pull request by dougmcneall**
This is my first pull request so let me know if I've messed anything up. I had problems converting R markdown files to markdown files using the code in the vignette style guide. This might need changing if it's crucial.

### CODE REVIEW

File: vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+title: "Sensitivity analysis of COVID cumulative cases at lockdown"
+author: "Doug McNeall"
+date: "2020-06-01"
+output: html_document
+---
+
+
+This is a preliminary sensitivity analysis to test data, emulators and techniques, and shouldn't be used to draw conclusions. There appears to be at least one design point where MetaWards "breaks", for example infecting more people by lockdown than there are in the population.
+
+We use a Gaussin process emulator from package DiceKriging to model the variation of cumulative infections at lockdown (23rd March 2020) with parameters. We then run a simple sensitivity analysis, and an initial history-matching type exercise, comparing the model with external estimates of cumulative cases from Jit et al. (2020).
```
Comment (sieste) [inline vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd]: Gaussian

File: vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+library(RColorBrewer)
+
+source("https://raw.githubusercontent.com/dougmcneall/packages-git/master/emtools.R")
+source("https://raw.githubusercontent.com/dougmcneall/packages-git/master/imptools.R")
+source("https://raw.githubusercontent.com/dougmcneall/packages-git/master/vistools.R")
+
+```
+
+
+Load the cumulative infections by day 79 (23rd March 2020) in local authorities This data has the parameters BUT NOT THE RUN NUMBER or unique ID.
```
Comment (sieste) [inline vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd]: period missing

File: vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+     main = 'leave-one-out cross validtaion',
+     pch = 19,
+     ylim = ylim)
+segments(x0 = y.national, y0 = loo$mean - (2*loo$sd), x1 = y.national, y1 = loo$mean + (2*loo$sd))
+legend('topleft', legend = "bars indicate \u00B12 sd", bty = 'n')
+abline(0,1)
+
+
+```
+
```
Comment (sieste) [inline vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd:162]: any conclusions?

File: vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+blues = brewer.pal(9, 'Blues')
+
+pairs(X.unif.kept.unnorm[, 1:3],
+      gap = 0,
+      panel = dfunc.up,
+      dfunc.col = blues
+      )
+```
+
+
```
Comment (sieste) [inline vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd:278]: maybe add a concluding paragraph?

**Review by sieste — CHANGES_REQUESTED** [review]
Hi Doug, Thanks for contributing the vignette. Could you quickly address my comments, then I'll merge your vignette so it shows on the website. Cheers, Stefan

**Comment by dougmcneall** [comment]
Hi Stefan, I changed the vignette according to your requests and pushed it back to the repo. Did I do it right? Can you see it to merge?

Cheers,

Doug

File: vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd (hunk 1 above)
Comment (dougmcneall) [reply vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd]: Sorted

File: vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd (hunk 2 above)
Comment (dougmcneall) [reply vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd]: Sorted.

File: vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd (hunk 4 above)
Comment (dougmcneall) [reply vignettes/cumulative_infections_at_lockdown_sensitivity.Rmd:278]: Added

**Comment by BayesExeter** [comment]
I see Stefan has been commenting so I'll post a quick reminder, otherwise I'll merge for you as it would be good to have these online. Maybe meeting Leon tomorrow.

**Comment by dougmcneall** [comment]
I pushed a new version yesterday but Stefan didn’t see it - I may have messed it up? The diffs look ok though. Can you confirm you can see both commits? Stefan hasn’t got back yet.

Doug

**Review by BayesExeter — CHANGES_REQUESTED** [review]
Hi Doug,
 
So looking closely, I think it's fine, but because other changes have been merged, there would be an issue. Can you pull the latest version of the vignettes.md (the one with the quantile emulator etc) and add to that instead, then recommit.

**Comment by dougmcneall** [comment]
Ah, ok, might have to do that in a little while (busy with kids) but will complete before tomorrow morning no problems.

**Comment by dougmcneall** [comment]
Would you mind trying a merge? It’s only a line in vignettes.md and I think it should work (it’ll just merge new stuff in?). I’ve looked at the instructions for updating at my end and it would be quicker just to do the whole thing again!

**Comment by BayesExeter** [comment]
> Would you mind trying a merge? It’s only a line in vignettes.md and I think it should work (it’ll just merge new stuff in?). I’ve looked at the instructions for updating at my end and it would be quicker just to do the whole thing again!
> […](#)

Can't you just correct vignettes.md by hand on GitHub? Take the current master branch copy, select all copy, then paste into your vignettes.md and correct your line. The diff file in your commit seems to suggest we'd lose the additions.

**Comment by dougmcneall** [comment]
OK, I've tried that, hope it works OK. If not I'll scrub and try the whole thing again.

**Comment by BayesExeter** [comment]
> OK, I've tried that, hope it works OK. If not I'll scrub and try the whole thing again.

Hi Doug,

I know I'm being a massive pain here, so sorry. The new vignettes.md file you have will not break the existing one :), but also doesn't link in the vignette you want to add. I don't know if you want to add a link or just change the sensitivity analysis link. I think it's fine after that.

**Comment by dougmcneall** [comment]
No, you're not being a pain - it's just me being stupid *facepalm*

**Comment by dougmcneall** [comment]
Right, line added in vignettes.md. Hope that does it (and I'm learning how this goes!)

**Review by BayesExeter — APPROVED** [review]
All looks good. Going to merge now and check the site.

**Pull request merged**